test_opengraph_standards.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import requests
+
+from bs4 import BeautifulSoup
+
+
+class test_open_graph_standards:
+
+    def test(self, meta, required_tags, found_tags):
```

Comment:
You are killing me, man. Change the name of your testmethod. Change it from test to something with more detail. Maybe test_open_graph_tags_present. Name the test method that. It's called test right now. Rename the method.